app/bundles/CoreBundle/Form/RequestTrait.php

```diff
@@ -121,7 +121,9 @@ protected function prepareParametersFromRequest(Form $form, array &$params, $ent
 
                     if (!$dateTest) {
                         // Date placeholder was used so just ignore it to allow import of the field
-                        unset($params[$name]);
+                        if (!defined('MAUTIC_API_REQUEST') || (defined('MAUTIC_API_REQUEST') && !MAUTIC_API_REQUEST)) {
```

Comment:
We have to find another way how to fix this. Using global constants makes the code untestable because you cannot change their value. You set them once and all tests must use this value and so you cannot test the other values of this costant.

Reply:
Then we can use static  method  isApiRequest https://github.com/mautic/mautic/blob/7a2875bc16a6de50a1803fbc41b64278ae9b5428/app/bundles/ApiBundle/Helper/RequestHelper.php#L23

Or do you any idea?

Reply:
Yes, let's use that.

Reply:
It make issue also with any overwriteWithBlank flag for date time fields. I fixed it in different way.